apps/site/lib/site_web/templates/event/_month.html.eex

```diff
@@ -3,20 +3,34 @@
   <%= if @conn.assigns.date.year == @year and @conn.assigns.date.month == @month_number do %>
 
     <!-- Previous months button. A JS script sets up a listener for button click -->
-    <li class="list-group-item">
-      <a tabindex="0" role="button" class="m-previous-events-button" data-group="<%= render_event_month_slug(@month_number, @year) %>">
-        View Previous <%= render_event_month(@month_number, @year) %> Events
-        <i class="fa fa-angle-down down" aria-hidden="true"></i>
-      </a>
-    </li>
-    <%= for event_teaser <- @event_teasers do %>
-      <%= render "_event_teaser.html",
-          event_teaser: event_teaser,
-          check_event_ended: true,
-          conn: @conn,
-          month_number: @month_number,
-          year: @year %>
-    <% end %>
+    <div class="m-view-previous-events hidden">
+      <li class="list-group-item">
+        <a tabindex="0" role="button" class="m-previous-events-button" data-group="<%= render_event_month_slug(@month_number, @year) %>">
+          View Previous <%= render_event_month(@month_number, @year) %> Events
+          <i class="fa fa-angle-down down" aria-hidden="true"></i>
+        </a>
+      </li>
+      <%= for event_teaser <- @event_teasers do %>
+        <%= render "_event_teaser.html",
+            event_teaser: event_teaser,
+            check_event_ended: true,
+            conn: @conn,
+            month_number: @month_number,
+            year: @year %>
+      <% end %>
+    </div>
+
+    <!-- Show all events for the month if JS is disabled (instead of the div.m-view-previous-events) -->
+    <noscript>
+      <%= for event_teaser <- @event_teasers do %>
+        <%= render "_event_teaser.html",
+            event_teaser: event_teaser,
+            check_event_ended: false,
+            conn: @conn,
+            month_number: @month_number,
+            year: @year %>
+      <% end %>
+    </noscript>
```

Comment:
Is there a way to simplify this noscript block with the duplicated else block below?